openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -206,20 +233,21 @@ def _validate_alchemical_components(
         raise ValueError(errmsg)
 
     # Check that all alchemical components are mapped & small molecules
-    mapped = {'stateA': [m.componentA for m in mapping],
-              'stateB': [m.componentB for m in mapping]}
+    mapped = {"stateA": [m.componentA for m in mapping], "stateB": [m.componentB for m in mapping]}
```

Comment:
This also falls under the category of "ruff seems to hate multiple lines even when it would be better for readability".